Library/Homebrew/test/tab_test.rb

```diff
@@ -241,15 +241,18 @@ def test_to_json
     assert_equal @tab.changed_files, tab.changed_files
     assert_equal @tab.tap, tab.tap
     assert_equal @tab.spec, tab.spec
-    assert_equal @tab.time, tab.time
+    assert_nil @tab.time
+    assert_nil tab.time
```

Comment:
Yes, but doesn't it just whine because it is checking equality for `nil`? If `@tab.time` wasn't nil, the warning would go away, wouldn't it?